File: ParadigmaComp/15Questoes/golang/questao10.go
package main
//Métodos com Nomes Diferentes 
import "fmt"

// Função para somar dois números
func SomarDois(a, b int) int {
	return a + b
}

// Função para somar três números
func SomarTres(a, b, c int) int {
	return a + b + c
}

func main() {
	fmt.Println(SomarDois(1, 2))       // Soma de dois números
	fmt.Println(SomarTres(1, 2, 3))    // Soma de três números
}

File: ParadigmaComp/15Questoes/python/questao10.py
#Métodos com Nomes Diferentes
class Calculadora:
    def somar(self, a, b):
        return a + b

    def somar_tres(self, a, b, c):
        return a + b + c

# Exemplo de uso
calc = Calculadora()
print(calc.somar(1, 2))       # Soma de dois números
print(calc.somar_tres(1, 2, 3))  # Soma de três números
